lib/src/value/calculation.dart

```diff
@@ -158,6 +305,167 @@ class SassCalculation extends Value {
     return SassCalculation._("clamp", args);
   }
 
+  /// Creates a `pow()` calculation with the given [arguments].
+  ///
+  /// Each argument must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  ///
+  /// This may be passed fewer than three arguments, but only if one of the
+  /// arguments is an unquoted `var()` string.
+  static Value pow(Object base, Object? exponent) {
+    _verifyLength([base, if (exponent != null) exponent], 2);
+    base = _simplify(base);
+    exponent = exponent.andThen(_simplify);
+    if (base is! SassNumber ||
+        exponent is! SassNumber ||
+        exponent.hasUnit('%') ||
+        base.hasUnit('%')) {
+      return SassCalculation._("pow", [base, exponent ?? double.nan]);
```

Comment:
`exponent` should default to the constant `e` here, not `NaN`.

Reply:
Got it. I created a get function from the numbers library to avoid importing the math library to this file.